workflows/autometa.sh

```diff
@@ -145,20 +176,164 @@ autometa-taxonomy-majority-vote --lca $lca --output $votes --dbdir $ncbi
+        if [ ! -f $kingdom_fasta ]
+        then
+            echo "${kingdom_fasta} does not exist, skipping..."
+            continue
+        fi
+
+        # Retrieve contig IDs from kingdom fasta file
+        set -x
+        grep ">" $kingdom_fasta | \
+            sed 's/^>//' | \
+            sed 's/$/_/' | \
+            cut -f1 -d" " > $contig_ids
+        # Retrieve ORF IDs from contig IDs
+        grep -f $contig_ids $orfs | sed 's/^>//' | cut -f1 -d" " > $orf_ids
+        # Retrieve ORF seqs from ORF IDs
+        seqkit grep \
+            --pattern-file $orf_ids \
+            --out-file $kingdom_orfs \
+            $orfs
+        # Concatenate kingdom ORFs to single file for GTDB blastp
+        cat $kingdom_orfs >> $gtdb_orfs
+        { set +x; } 2>/dev/null
+    done
+    dbtype="gtdb"
+    prefix="${simple_name}.${dbtype}"
+    
+    # Step 5.2: Run blastp
+    # input:
+    # $gtdb_orfs --> Generated from step 5.1
+    gtdb_dmnd_db="${gtdb}/gtdb.dmnd" # generated using autometa-setup-gtdb (Must be performed prior to using this script)
+    # output
+    blast="${outdir}/${prefix}.blastp.tsv"
+
+    # script
+    set -x
+    diamond blastp \
+        --query $gtdb_orfs \
+        --db $gtdb_dmnd_db \
+        --evalue 1e-5 \
+        --max-target-seqs 200 \
+        --threads $cpus \
+        --outfmt 6 \
+        --out $blast
+    { set +x; } 2>/dev/null
+
+    #Step 5.3: Determine LCA
+    # input:
+    # $blast --> Generated from step 5.2
+    # $gtdb --> User Input
+    # $dbtype --> Updated according to $taxa_routine
+    dbtype="gtdb"
+
+    # output:
+    lca="${outdir}/${prefix}.orfs.lca.tsv"
+    sseqid_to_taxid="${outdir}/${prefix}.orfs.sseqid2taxid.tsv"
+    error_taxids="${outdir}/${prefix}.orfs.errortaxids.tsv"
+
+    # script:
+    set -x
+    autometa-taxonomy-lca \
+        --blast $blast \
+        --dbdir $gtdb \
+        --dbtype $dbtype \
+        --lca-output $lca \
+        --sseqid2taxid-output $sseqid_to_taxid \
+        --lca-error-taxids $error_taxids 
+    { set +x; } 2>/dev/null
+
+    # Step 5.4: Perform Modified Majority vote of ORF LCAs for all contigs that returned hits in blast search
+    # input:
+    # $lca --> Generated from step 5.3
+    # $gtdb --> User Input
+
+    # output:
+    votes="${outdir}/${prefix}.taxids.tsv"
+
+    # script:
+    set -x
+    autometa-taxonomy-majority-vote \
+        --lca $lca \
+        --output $votes \
+        --dbdir $gtdb \
+        --dbtype gtdb
+    { set +x; } 2>/dev/null
+
+    # Step 5.5: Split assigned taxonomies into kingdoms
+    # input:
+    # $votes --> Generated from step 5.4
+    # $outdir --> Generated from step 0
+    # prefix="${prefix}.${dbtype}"
+    # $filtered_assembly --> Generated from step 1
+    # $gtdb --> User Input
+
+    # output:
+    # Will write recovered superkingdoms to $outdir
+    # e.g. ${outdir}/${prefix}.gtdb.bacteria.fna
+    # e.g. ${outdir}/${prefix}.gtdb.archaea.fna
+    # e.g. ${outdir}/${prefix}.gtdb.taxonomy.tsv
```

Comment:
`gtdb`, i.e. `$dbtype` is included in `$prefix`

```suggestion
    # e.g. ${outdir}/${prefix}.bacteria.fna
    # e.g. ${outdir}/${prefix}.archaea.fna
    # e.g. ${outdir}/${prefix}.taxonomy.tsv
```